dart/lib/src/sentry_options.dart

```diff
@@ -27,6 +27,17 @@ class SentryOptions {
   /// used, the SDK will not send any events.
   String? dsn;
 
+  /// Evaluates and parses the DSN. May throw an exception if the DSN is invalid.
+  @internal
+  late final Dsn parsedDsn = _parseDsn();
```

Comment:
the dsn could change in theory, even if not supported.
I'm thinking about the case when a user reinits the SDK with the same options changing the dsn. We would still have the old parsedDsn.

Let's reevaluate it when the dsn is set?
wdyt?

Reply:
👍 updated it